README.md

```diff
@@ -100,6 +100,26 @@ $ docker volume rm MyVolume
 $ /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py volume ls
 ```
 
+## Upgrade Instructions
```

Comment:
1. This section should be after installation section (https://github.com/ashahi1/docker-volume-vsphere/blob/198f25c61af6948ccc4d454c3db09eec909b7970/README.md#installation-instructions)

2. content should be part of Installation.md (https://github.com/ashahi1/docker-volume-vsphere/blob/master/docs/user-guide/install.md) and have a reference on this page.